src/frontend/src/handler/create_function.rs

```diff
@@ -48,7 +51,7 @@ pub async fn handle_create_function(
         Some(lang) => {
             let lang = lang.real_value().to_lowercase();
             match &*lang {
-                "python" | "java" => lang,
+                "python" | "java" | "wasm_v1" => lang,
```

Comment:
The version can be encoded as metadata in wasm binaries so that the language can be always "wasm".